src/assets/locales/fr.json

```diff
@@ -451,6 +512,13 @@
   "safari-mic-modal.message": "<p>Hubs nécessite les permissions du micro dans Safari.</p><p>Pour continuer, merci de recharger la page et d'autoriser l'accès au micro.</p>",
   "safari-mic-modal.reload-button": "Recharger",
   "safari-mic-modal.title": "Accès au micro requis",
+  "save-api-token.copied": "Copié",
+  "save-api-token.copy-label": "Jeton API",
+  "save-api-token.description1": "Veuillez sauvegarder ce jeton API dans un endroit sûr.",
+  "save-api-token.description2": "Vous ne serez pas en mesure de le voir de nouveau une fois que vous aurez fermé cette fenêtre.",
+  "save-api-token.revoke": "Confirmer et fermer",
+  "save-api-token.scopes": "Portées d'accès",
+  "save-api-token.title": "Jeton API",
```

Comment:
"save-api-token.copied": "Copied",
  "save-api-token.copy-label": "API Token",
  "save-api-token.description1": "Please save this API token in a safe place.",
  "save-api-token.description2": "You will not be able to see it again once you have closed this window.",
  "save-api-token.revoke": "Confirm and close",
  "save-api-token.scopes": "Scopes",
  "save-api-token.title": "API Token",

Reply:
```suggestion
  "save-api-token.title": "Jeton d'API",
```